www.example.com/db/openid-store/associations/http-localhost-Sve98kiqQxHfzN8CdZK0Uuu1aIs-OgWy9gaq1b8EEKWdb1vzzoUouUY

```diff
@@ -0,0 +1,6 @@
+version:2
+handle:{HMAC-SHA1}{5b22bd47}{xQKqRw==}
```

Comment:
Hi, I think this may have been added by accident... can you remove it? 

https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

Reply:
And the other similar files... i think they're not connected to the issue, right? Thanks!